webviz_subsurface/plugins/_vfp_analysis/_plugin.py

```diff
@@ -0,0 +1,52 @@
+from typing import Callable, Dict, List, Optional, Tuple
+
+from webviz_config import WebvizPluginABC, WebvizSettings
+from webviz_config.utils import StrEnum
+
+from ._utils import VfpDataModel
+from ._views._vfp_view import VfpView
+
+
+class VfpAnalysis(WebvizPluginABC):
+    """Vizualizes VFP curves.
+
+    ---
+
+    * **`vfp_file_pattern`:** File pattern for where to search for vfp arrow files. The path
+    should be relative to the runpath if ensemble and realization is given as input, if not
+    the path needs to be absolute.
+    * **`ensemble`:** Which ensemble in `shared_settings` to use..
+    * **`realization`:** Which realization to pick from the ensemble.
```

Comment:
In case the given realization has failed, this plugin will also then fail? Maybe ok given the speed increase you get with looking at a particular realization?

Reply:
Is it better to give a warning and open the app without any VFP curves? It currently will give a `FileNotFoundError` if the ensemble exists, but not the realization.

Reply:
I think current approach with error instead of warning is preferred.

One alternative is perhaps to omit realization argument and instead use the first realization that succeeds (since we only take one realization we already assume it is constant across the ensemble)? But that has the downside of being less explicit... 🤔

I'm ok with both - you can choose @lindjoha 🙂

Reply:
I think I'll just go for the current solution.